packages/components/organisms/f-footer/test/specs/component/shared/f-footer.component.spec.js

```diff
@@ -106,8 +106,23 @@ describe('Shared - f-footer component tests', () => {
         expect(footer.isCurrentCountryIconDisplayed(expectedLocale)).toBe(true);
     });
 
-    forEach([['au', 'au'], ['at', 'at'], ['be', 'be-en'], ['bg', 'bg'], ['ca_en', 'skipthedishes.com'], ['ca_fr', 'skipthedishes.com/fr'], ['dk', '.dk'], ['jet_fr', '.fr'], ['de', '.de'], ['ie', '.ie'], ['il', '.il'], ['it', '.it'],
-        ['lu', 'lu-en'], ['nl', '.nl'], ['nz', '.nz'], ['no', '.no'], ['pl', '.pl'], ['pt', '/pt'], ['ro', '/ro'], ['es', '.es'], ['ch_ch', '.ch'], ['ch_en', '/en'], ['ch_fr', '/fr']])
+    forEach([['au', 'au'], ['at', 'at'], ['no', 'no'], ['dk', 'dk'], ['be', 'be-en'], ['bg', 'bg'], ['ca_en', 'skipthedishes.com'], ['jet_fr', '.fr'], ['de', '.de'], ['ie', '.ie'], ['il', '.il'], ['it', '.it']])
+    .it('should display all countries and redirect to correct URL', (country, expectedUrl) => { 
```

Comment:
I know this is the message signature as it existed previously, but might be good to use the [variable placeholders](https://www.npmjs.com/package/mocha-each#string-title) to insert the test case being run, to make failing cases easier to identify. Additionally, I for one would find it easier to parse if each case is on an individual line. e.g:

```suggestion
    forEach([
      ['au', 'au'],
      ['at', 'at'],
      ['no', 'no'],
      ['dk', 'dk'],
      ['be', 'be-en'],
      ['bg', 'bg'],
      ['ca_en', 'skipthedishes.com'],
      ['jet_fr', '.fr'],
      ['de', '.de'],
      ['ie', '.ie'],
      ['il', '.il'],
      ['it', '.it']
    ]).it('should display link for country code "%s" and redirect to correct URL', (country, expectedUrl) => { 
```

Reply:
ooh this is awesome, thank you! We were discussing this yesterday in team web testers actually - I think some of these lengthy forEach's will end up going, but this is great to implement in the meantime and for the smaller iterable tests! 👍